CheckerServer/CheckerServer/Models/Order.cs

```diff
@@ -7,24 +7,24 @@ public class Order : BaseDBItem
         [ForeignKey("Restaurant")]
         public int RestaurantId { get; set; }
         public int Table { get; set; }
-        public OrderStatus Status { get; set; } = OrderStatus.Ordered;
+        public eOrderStatus Status { get; set; } = eOrderStatus.Ordered;
         public List<OrderItem> Items { get; set; } = new List<OrderItem>();
-        public OrderType OrderType { get; set; } = OrderType.FIFO;
+        public eOrderType OrderType { get; set; } = eOrderType.FIFO;
         public float TotalCost { get; set; }
         public float RemainsToPay { get; set; }
     }
 
-    public enum OrderStatus
```

Comment:
i don't know if that's matter but all my enums are on a separate directory

Reply:
It doesn't matter, in this case.